src/DotVVM.Framework/Hosting/DotvvmPresenter.cs

```diff
@@ -249,6 +262,14 @@ public class DotvvmPresenter : IDotvvmPresenter
             }
         }
 
+        private long addTraceData(long lastStopwatchState, string processName, IDotvvmRequestContext context)
```

Comment:
Please rename the method to `AddTraceData`.
And the rename the `processName` to `eventName`.